noxfile.py

```diff
@@ -132,11 +133,11 @@ def get_sphinx_build_command(kind):
         return [
             "sphinx-build",
             "-W",
-            "-c", "docs/html",  # see note above
-            "-d", "docs/build/doctrees/" + kind,
-            "-b", kind,
-            "docs/" + kind,
-            "docs/build/" + kind,
+            "-c=docs/html",  # see note above
+            f"-d=docs/build/doctrees/{kind}",
+            f"-b={kind}",
```

Comment:
Obligatory grumpy comment about black 🙂 

On principle, I don't like the way these three merge two arguments just because black can't tell that they are logically linked, and would make this call less readable.

I'd prefer to see this written with `fmt: off` and paired arguments on one line, because it's more readable, and more robust, that way.

Reply:
🤷🏽 

Works either way for me -- I'll add the `# fmt: off` here. :)